README.md

```diff
@@ -1,28 +1,47 @@
-![R-CMD](https://github.com/JGCRI/hector/workflows/R-CMD/badge.svg)
-![Command Line Hector](https://github.com/JGCRI/hector/workflows/Command%20Line%20Hector/badge.svg)
-![test-coverage](https://github.com/JGCRI/hector/workflows/test-coverage/badge.svg)
-[![codecov](https://codecov.io/gh/JGCRI/hector/branch/master/graph/badge.svg?token=EGM0lXDxRv)](https://codecov.io/gh/JGCRI/hector)
-![lint](https://github.com/JGCRI/hector/workflows/lint/badge.svg)
 
 Hector
 ======
```

Comment:
@leeyap and @bpbond this could use some updating and more details. @crvernon format that JGCRI tools would like to use for our read mes?

Reply:
I get that, but why remove the badges?

Reply:
I thought that I moved them to below the Hector name cause I thought it kinda looked weird to have the badges over the title